### DISCUSSION TITLE
Please help me configure the Crazyflie V2.1 using the client.

### DISCUSSION THREAD

**Discussion by Zm442709994** [bitcraze/discussions#659] [Q&A] [answered]
In the mode of lighthouse positioning：

First, currently when using the client to control the drone, after pressing a point, the drone travels approximately 50 centimeters. How can I set it to travel a distance of 10 centimeters or less?

Second, currently when using the client to control the drone's flight, the yaw angle remains fixed. How can I use the client to control and change the drone's yaw angle?

How to realize the flight of unmanned aerial vehicle along the preset trajectory in the end?

Does anyone know how to configure it?Much appreciated!

**Comment by ataffanel** [comment, marked as answer]
Hi,

Now that you got the system working with the client, the next step will be to write python scripts for autonomous flight: while it would be possible to modify the client to control more complex autonomous flight, this is not something the client has been designed for so it would not be an easy task.

I suggest you start by this guide to learn how the crazyflie python lib works and how to connect and communicate with the Crazyflie: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/

Then you will be able to understand and adapt to your need one of the lib example. The examples in the autonomy folder are likely the one you will be interested about, they are controlling autonomous flight using different APIs (from [sending simple setpoint at regular interval](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/autonomy/autonomousSequence.py) to [following trajectory](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/autonomy/autonomous_sequence_high_level_compressed.py)): https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/autonomy.
